requirements.txt

```diff
@@ -15,3 +15,4 @@ opt_einsum~=3.3
 requests~=2.31.0
 typing_extensions~=4.5.0
 tomli~=2.0.0 # Drop once minimum Python version is 3.11
+tach~=0.13.1
```

Comment:
Same comment here. Out of scope for this PR, but it really makes no sense to me to have a `requirements.txt` file and then not use it in `setup.py`.

Reply:
requirements.txt should never be used in setup.py
They have completely different use cases. https://packaging.python.org/en/latest/discussions/install-requires-vs-requirements/#requirements-files

One specifies needs for the package, the other specifies a full execution environment.